src/PersonObjects/Player/PlayerObjectGeneralMethods.ts

```diff
@@ -603,6 +603,11 @@ export function focusPenalty(this: PlayerObject): number {
 }
 
 /** This doesn't change the current page; that is up to the caller. */
+/**
+ * WIP: What do you think about renaming this function to setInfiltration? "setInfiltration" is exactly what this
```

Comment:
In my follow-up I am also setting the player's location, so I don't think setInfiltration is the right name. 

You are right that "startInfiltration" is overloaded, though. There is this broader type of start, where we bring up the UI and assign the class, and then the more specific type of the minigames starting. 

Maybe `initInfiltration` is the right name?